source/Mlos.Python/mlos/Optimizers/OptimizerBase.py

```diff
@@ -56,15 +56,29 @@ def predict(self, feature_values_pandas_frame, t=None) -> Prediction:
         """
         raise NotImplementedError("All subclasses must implement this method.")
 
-    @abstractmethod
-    def optimum(self, stay_focused=False) -> Dict: # TODO: make it return an object

```

Comment:
Is the "TODO: make it return an object" comment still worth keeping?

Reply:
Hm probably not as it's now returning two points?

Reply:
I noted down below that it may make more sense to return named params as well, but that can certainly get pushed off to a different PR.

Reply:
I think you are addressing the TODO with this PR so I think it's good that you changed it. I would also change the type hint of the return value to: ```-> Tuple[Point, Point]```